examples/linear_model/plot_nnls.py

```diff
@@ -0,0 +1,59 @@
+"""
+==========================
+Non-negative least squares
+==========================
+
+In this example, we fit a linear model with positive constraints on the
+regression parameters and compare the estimated parameters to a classic linear
```

Comment:
Parameter is a bit overloaded so I'd suggest coefficient which we more commonly use

```suggestion
regression coefficients and compare the estimated coefficients to a classic linear
```